Chrome/manifest.json

```diff
@@ -1,43 +1,59 @@
 {
-    "manifest_version": 3,
-    "name": "MoodleCustomizer",
-    "version": "1.1.1",
-    "author": "Hokkaydo",
-    "description": "Improve your Moodle experience.",
-    "icons": {
-        "16": "icons/logo.png"
-    },
-    "permissions": [
-        "activeTab",
-        "tabs",
-        "webNavigation",
-        "scripting",
-        "storage"
-    ],
-    "host_permissions": [
-        "<all_urls>"
-    ],
-    "content_scripts": [
-        {
-            "matches": [
-                "<all_urls>"
-            ],
-            "js": [
-                "index.js"
-            ]
-        }
-    ],
-    "action": {
-        "default_title": "On/Off",
-        "default_popup": "popup/options.html"
-    },
-    "background": {
-        "service_worker": "background.js"
-    },
-    "web_accessible_resources":[
-        {
-            "resources": ["retrieveVariables.js"],
-            "matches": ["<all_urls>"]
-        }
+  "manifest_version": 3,
+  "name": "MoodleCustomizer",
+  "version": "1.1.2",
+  "author": "Hokkaydo",
+  "description": "Improve your Moodle experience.",
+  "icons": {
+    "16": "icons/logo.png"
+  },
+  "permissions": [
+    "activeTab",
+    "tabs",
+    "webNavigation",
+    "scripting",
+    "storage",
+    "declarativeNetRequest",
+    "declarativeNetRequestWithHostAccess"
+  ],
+  "host_permissions": [
+    "<all_urls>"
+  ],
+  "content_scripts": [
+    {
+      "matches": [
+        "http://*/*",
```

Comment:
Like you say, it was pain in the ass to make it work under Chrome, particulary to match urls. 
Does it work like you wrote ? I remember it was hard for me to find the right string to set due to poor doc.
But if it works, it's fine

Reply:
I changed this one because otherwise i had errors with the extension trying to inject a script in chrome:// URL

(When you install the extension, you made a script to install it on every opened tab, including the chrome extensions settings tab,  which produce an error) 

But I didn’t try again the script injection after this change, but i will

Reply:
https://developer.chrome.com/docs/extensions/mv3/match_patterns/
should work according to this

Reply:
According to doc, develop Chrome extension should be easy but that's not a fact.
Could you check that this change didn't broke the extension on Chrome ?